contrib/mesos/pkg/scheduler/podtask/pod_task.go

```diff
@@ -33,8 +33,8 @@ import (
 )
 
 const (
-	DefaultContainerCpus = 0.25 // initial CPU allocated for executor
-	DefaultContainerMem  = 64   // initial MB of memory allocated for executor
+	DefaultContainerCpus = 0.25                   // initial CPU allocated for executor
+	DefaultContainerMem  = 64.0 * 1024.0 * 1024.0 // initial MB of memory allocated for executor
```

Comment:
K8s uses bytes everywhere. To reuse the limit extraction code I do this as well everywhere and divide by 1024^2 when comparing to the offer values.